api/metrics-api-jaxrs/pom.xml

```diff
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hawkular-metrics-parent</artifactId>
     <groupId>org.hawkular.metrics</groupId>
-    <version>0.3.0-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
On 10 Mar 2015, at 18:15, Thomas Segismont wrote:

>
> I disagree. If projects depend on SNAPSHOTS it's because they want the 
> latest thing, should it break their code over night. If they want more 
> stability, then let's release 0.3.0 and then bump to 0.3.1-SNASPHOT

I think here making a huge api change just to have other consumers fix 
their code out of the blue is wrong.
So +1 on release 0.3.0 and then apply the change to 0.3.1, so consumers 
can use 0.3.0 while adapting their
code to those changes.


-- 
hwr@pilhuhn.de  m:0179/207 4919  b:http://pilhuhn.blogspot.com